README.md

```diff
@@ -39,27 +57,32 @@ Arguments:
 >
 >  --nameMap NAMEMAP     Either a file mapping generated pathway fileNames to parameter values, "stripped" to exclude the directory and extension from the filename, or "fileName" to use raw file names. Optional, default = stripped.
 >
+>  --saveGraphlets       If set, will save graphlet distributions as pickled dictionaries.
+>
 >  --verbose             If set, will print intermediate status updates. Optional, default = False.
 
-## Example
+## Examples
 
-`scripts/runPPA.sh` runs pathway parameter advising on any set of sif or edgelist networks.
+`bin/runPPA.sh` runs pathway parameter advising on any set of sif or edgelist networks.
 It will attempt to install the pgd library and setup the reactome pathway graphlets automatically. 
```

Comment:
`runPPA.sh` no longer installs PGD

Reply:
Removed this where runPPA.sh is mentioned.